### PR TITLE
feat(lib-lesson)[skp-262]: Creates lesson feature page

### PR DESCRIPTION
**Pull request by Gardemarin**

### CODE REVIEW

File: apps/ppm/src/app/routes/private.ts [hunk 1]
```diff
@@ -53,6 +54,13 @@ const PrivateRouter: RouterItem[] = [
     component: FeaturesAdvertForm,
     onMenu: false,
   },
+  {
+    path: `/${PrivateRoutesPath.LESSON}${PrivateRoutesPath.GET_BY_ID}`,
+    title: 'Lesson',
+    icon: RouterMenuIcons.MENTOR,
```
Comment (Neutronas) [inline apps/ppm/src/app/routes/private.ts:60]: Why mentor icon on lessons page?

File: libs/features/advert-page/src/lib/features-advert-page.tsx [hunk 2]
```diff
@@ -71,6 +72,7 @@ let lessons = [
 let lessonsDescription =
   'Are you ready to find out what all the hype is about with ReactJS? These ReactJS for beginners tutorials will bring you completely up to speed on the hottest JavaScript framework used on the web today.';
 
+const accordionTitle = 'Lessons';
```
Comment (Neutronas) [inline libs/features/advert-page/src/lib/features-advert-page.tsx:75]: If title of accordion is hardcoded, why do we need to assign it to variable? Just provide string directly to component.

**Review by Neutronas — CHANGES_REQUESTED** [review]

File: libs/features/advert-page/src/lib/features-advert-page.tsx (hunk 2 above)
Comment (Gardemarin) [reply libs/features/advert-page/src/lib/features-advert-page.tsx:75]: It allows easier to reuse the same shared component. For example, you use it as "Lessons", but I use it as "Related lessons".

File: apps/ppm/src/app/routes/private.ts (hunk 1 above)
Comment (Gardemarin) [reply apps/ppm/src/app/routes/private.ts:60]: I Did not know what else I could to use. So I used this value.

Comment (doncevas) [reply apps/ppm/src/app/routes/private.ts:60]: if you use     onMenu: false icon should be null

**Review by doncevas — CHANGES_REQUESTED** [review]

File: libs/features/lesson-page/src/lib/features-lesson-page.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    id: string;
+  };
+  path?: string | string[];
+}
+
+/* eslint-disable-next-line */
+export interface FeaturesLessonPageProps {}
+
+export const FeaturesLessonPage = (props: {history: History, match: RouteComponentProps<RouteInfo>}) => {
+  const lessonsDescription = 'Are you ready to find out what all the hype is about with ReactJS? These ReactJS for beginners tutorials will bring you completely up to speed on the hottest JavaScript framework used on the web today.';
```
Comment (doncevas) [inline libs/features/lesson-page/src/lib/features-lesson-page.tsx:21]: Add TODO comment if leaving static text

**Review by doncevas — CHANGES_REQUESTED** [review]

File: libs/features/lesson-page/src/lib/features-lesson-page.tsx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+interface RouteInfo extends RouteProps {
+  params: {
+    id: string;
+  };
+  path?: string | string[];
+}
+
+/* eslint-disable-next-line */
+export interface FeaturesLessonPageProps {}
```
Comment (doncevas) [inline libs/features/lesson-page/src/lib/features-lesson-page.tsx:18]: remove if useless

**Review by doncevas — CHANGES_REQUESTED** [review]

**Review by doncevas — APPROVED** [review]

**Review by Neutronas — APPROVED** [review]

**Pull request merged**